packages/experimental/navigation/otel/hooks/useTrace.ts

```diff
@@ -0,0 +1,39 @@
+import {trace, Tracer, TracerProvider} from '@opentelemetry/api';
+import {MutableRefObject, useEffect, useRef} from 'react';
+
+const TRACER_DEFAULT = {
+  name: 'default',
+  version: '1.0',
+};
+
+interface ConfigArgs {
+  name?: string;
+  version?: string;
+}
+
+type TracerRef = MutableRefObject<Tracer | null>;
+
+const useTrace = (
```

Comment:
minor, I think `useTracer` would be slightly more consistent with the other naming